compiler/tests/evm_contracts.rs

```diff
@@ -266,6 +267,7 @@ fn test_assert() {
     case("return_u256_from_called_fn.fe", vec![], Some(u256_token(42))),
     case("return_u256.fe", vec![], Some(u256_token(42))),
     case("return_identity_u256.fe", vec![42], Some(u256_token(42))),
+    case("return_identity_u128.fe", vec![42], Some(u256_token(42))),
```

Comment:
@g-r-a-n-t 
This test passes but I'm not 100 % sure if I'm testing the right thing because the inputs (`vec![42]`) are converted to `u256token`

```rust
input
                .clone()
                .into_iter()
                .map(|val| u256_token(val))
                .collect(),
```

I thought I need to create a `u128_token` variant of that but `ethabi::Token::Uint` simply expects a `U256` so maybe it's just something I don't understand :man_shrugging:

Reply:
The ABI encoding for uints is the same for every size, so it shouldn't really matter in practice how we encode uints for these tests. We might just want to rename `u256_token` to `uint_token`.

The difference in sizes of uints should only really be relevant internally when it comes to [packing](https://github.com/ethereum/fe/issues/129) values in memory and storage more efficiently.